### DISCUSSION TITLE
这2万人star没有人讨论一下吗

### DISCUSSION THREAD

**Discussion by huangjc0715** [bannedbook/fanqiang#671] [General]
这2万人star没有人讨论一下吗

**Comment by bannedbook** [comment]
![image](https://user-images.githubusercontent.com/4361923/124901646-c4c2ca80-e014-11eb-8b85-bfe354b5e0e8.png)

**Reply by huangjc0715** [reply]
哇，被作者回复了！等我变强大了也来帮你维护优化
